client/signup/config/steps-pure.js

```diff
@@ -445,6 +446,7 @@ export function generateSteps( {
 				'siteUrl',
 				'lastDomainSearched',
 				'useThemeHeadstart',
+				'domainCart',
```

Comment:
Related to this issue: p1696370532128529/1696279559.896049-slack-CKZHG0QCR

I tried to track down the problem, but ultimately was able to "fix it" by doing this. But, I'm not sure this is the best way to fix this. :/

Tangetially related, I think it would cool if we had a prop that signaled we wanted to enable multi-domain selection support or not.

```suggestion
				...( config.isEnabled( 'domains/add-multiple-domains-to-cart' ) ? [ 'domainCart' ] : [] ),
```

Reply:
Is it better to create a new step? (copy paste and iterate)

Not sure we want to be making multi domain selection available everywhere anyway. (Isn't the domain step used in other flows?)